src/test/resources/utils/dt-with-extends.json

```diff
@@ -2,7 +2,10 @@
   "@context": "dtmi:dtdl:context;2",
   "@id": "dtmi:com:cosmotech:supply:TransportOperation;1",
   "@type": "Interface",
-  "extends": "dtmi:com:cosmotech:supply:Operation;1",
```

Comment:
This remains a valid form for `extends`. So I suggest keeping this as is, so it is tested, but add your changes in a separate file and test case.

Reply:
done